From cd12ed2337965e75d6861a79925eef3601041126 Mon Sep 17 00:00:00 2001
From: Chris Williamson <chris@chrisaw.io>
Date: Sun, 22 Feb 2026 16:56:33 +0000
Subject: [PATCH 3/5] Remove ro.adb.secure=0 from system.prop

The AOSP build system already sets ro.adb.secure based on the build
variant. Having it in system.prop causes duplicate sysprop assignment
errors during build. The securize tweak in phh-on-boot.sh already
handles setting ro.adb.secure=1 at runtime.
---
 base.mk     | 1 -
 system.prop | 1 -
 2 files changed, 2 deletions(-)

diff --git a/base.mk b/base.mk
index f59cae3..37f6d38 100644
--- a/base.mk
+++ b/base.mk
@@ -37,7 +37,6 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
     ro.build.version.all_codenames=$(PLATFORM_VERSION_ALL_CODENAMES) \
     ro.build.version.release=$(PLATFORM_VERSION) \
     ro.build.version.security_patch=$(PLATFORM_SECURITY_PATCH) \
-    ro.adb.secure=0 \
     ro.logd.auditd=true \
     ro.logd.kernel=true \
 
diff --git a/system.prop b/system.prop
index f3c0c45..b2bcf63 100644
--- a/system.prop
+++ b/system.prop
@@ -1,4 +1,3 @@
-ro.adb.secure=0
 #Some devices have sdcardfs kernel panicing on 8.0
 #Disable for everyone for the moment
 ro.sys.sdcardfs=0
-- 
2.52.0

